views/postgres/index.erb

```diff
@@ -59,6 +60,13 @@
                   <%= pg[:name] %>
                 </a>
               </td>
+              <td class="whitespace-nowrap px-3 py-4 text-sm text-gray-500">
+                <% (partner, _, _) = flavors[pg[:flavor]] %>
```

Comment:
Maybe use some other name for this variable instead of partner, as Ubicloud is not technically a partner, but it is in this array.

Reply:
I renamed it as `brand`